src/io/opendss.jl

```diff
@@ -41,9 +51,9 @@ function create_starbus(tppm_data::Dict, transformer::Dict)::Dict
     starbus["bus_type"] = 1
     starbus["index"] = starbus_id
 
-    nodes_all = .+([parse_busname(transformer["buses"][n])[2] for n in length(transformer["buses"])]...)
-    active_phases = join(["$n" for n in 1:nconductors if nodes_all[n] > 0], ".")
-    starbus["source_id"] = "transformer.$(transformer["name"]).$(active_phases)"
+    nodes = .+([parse_busname(transformer["buses"][n])[2] for n in length(transformer["buses"])]...)
+    starbus["active_phases"] = [nodes[n] > 0 ? 1 : 0 for n in 1:nconductors]
```

Comment:
Instead of a vector of booleans, how about a sparse set of ints, like `Set{Int}([2,3])`.

Later on we can then do things like, `for p in comp["active_phases"] println(pg[p]) end`.

Reply:
my only concern with using Sets is that they seem to json serialize into lists. If you don't think that would be a problem, then it seems fine with me.

Reply:
In the PowerModels data dict we can use a sparse array (i.e. `[2,3]`) for JSON compatibility.  In the struct we can use a Set because it is already a native Julia type.